elastic/kibana
# [Upgrade Assistant] Fix small issue with unknown indicators

Closes https://github.com/elastic/kibana/issues/186247

## Summary

The ES api can sometimes return an indicator that has status unknown and that only has a symptom (no details). For example: `disk: { status: 'unknown', symptom: 'No disk usage data.' }`. This PR makes sure that the UI doesnt break in this specific scenario and creates a warning indicator anyway with the minimal information that was provided by the API.

##### How to test

* Start up kibana and es
* Run the following command in console:
```
 PUT _cluster/settings
 {
     "persistent": {
         "health.node.enabled": false
     }
 }
```
* Navigate to Stack management -> Upgrade assistant
* Verify that:
  * ES deprecations box shows one warning
  * Clicking on it goes to the `es_deprecations` page and a `No disk usage` deprecation is shown.
